pages/dashboard/index.js

```diff
@@ -173,39 +184,42 @@ const redirectToHome = {
   },
 };
 
-export const getServerSideProps = async ({ req, locale }) => {
-  const apolloClient = initializeApollo();
-  await apolloClient.query({
-    query: GET_USERS,
-  });
-
-  const session = await getSession({ req });
+export const getServerSideProps = async (context) => {
+  const session = await getSession(context);
   if (!session) {
+    console.error('Log in first!');
     return redirectToHome;
   }
 
-  await dbConnect();
+  const { req, locale } = context;
```

Comment:
can you just deconstruct in the function's parameters? Code will be more consistent with existing code and easier to follow.

Reply:
I tried doing that, getSession seems to only take in the entire context object, throwing in req did not work

Reply:
ohh I see. Didn't catch that, thanks!